vendor.conf

```diff
@@ -37,11 +37,18 @@ github.com/opencontainers/runc v1.0.0-rc2
 # OCI runtime spec, latest release as of 12/16/2016
 github.com/opencontainers/runtime-spec v1.0.0-rc3
 # logrus, latest release as of 12/16/2016
+# FIXME: github.com/docker/docker/pkg/archive requires this pkg as github.com/Sirupsen/logrus.
+# So we vendor github.com/Sirupsen as well.
```

Comment:
@crosbymichael  opened https://github.com/crosbymichael/go-runc/pull/5, which replaces sirupsen with Sirupsen.

Another idea is to replace Sirupsen with sirupsen, but it would require modifying many files in github.com/docker/docker

So I suggest replacing sirupsen with Sirupsen.